### PR TITLE
backstage: Remove all routes which do not begin with '__/origami/...'

### PR DESCRIPTION
**Pull request by notlee**
Our CDN has been updated and these are no longer needed.
https://github.com/Financial-Times/origami-build-service/pull/576

This commit also updates OBS v1 to redirect to v3 rather than v2.

### CODE REVIEW

File: lib/routes/v3/bundles.js [hunk 1]
```diff
@@ -4,13 +4,11 @@ const {createJavaScriptBundle} = require('../../middleware/v3/createJavaScriptBu
 const {createCssBundle} = require('../../middleware/v3/createCssBundle');
 module.exports = app => {
 	app.get([
-		'/v3/bundles/js/',
```
Comment (akomiqaia) [inline lib/routes/v3/bundles.js:7]: I am not sure but did you intend to remove v3 url?

Comment (akomiqaia) [reply lib/routes/v3/bundles.js:7]: I guess you did since this kind of changes are everywhere

**Review by akomiqaia — APPROVED** [review]

File: lib/routes/v3/bundles.js (hunk 1 above)
Comment (notlee) [reply lib/routes/v3/bundles.js:7]: Yeah, our CDN has been updated so these are no longer needed. The app will responsed to `'/__origami/service/build/v3/bundles/js/` still.
https://github.com/Financial-Times/origami-build-service/pull/576

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v366.1.0 :tada:

**Pull request merged**